src/ess/spectroscopy/indirect/time_of_flight.py

```diff
@@ -104,7 +61,6 @@ def detector_time_of_flight_data(
     )
     # This is time-of-flight at the sample.
     result.bins.coords['sample_tof'] = result.bins.coords.pop('tof')
-    del result.bins.coords['event_time_offset']
     del result.bins.coords['event_time_zero']
```

Comment:
unrelated to this PR, but I kept the `event_time_zero` coord in `essreduce` in case it was needed for some event filtering, based on sample environment conditions or other?